app/models.py

```diff
@@ -635,8 +635,6 @@ def get_reply_to_text(self):
         if self.template_type == LETTER_TYPE:
```

Comment:
think this'd be a bit cleaner if it were combined in to one line. It's a minor thing, but it'd mean that we'd actually hit the `return None` at the end of the if/elif branches, rather than implicitly returning None.
`if self.template_type == LETTER_TYPE and self.service_letter_contact_id is not None:`